src/main/xar-resources/data/lucene/lucene.xml

```diff
@@ -621,6 +620,8 @@
                         </variablelist>
                         <para>For instance:</para>
                         <programlisting language="xquery" xlink:href="listings/listing-41.txt"/>
+                        <para>To optimize performance you can also specify a minium number of matches to cut-off needless disjunctive searches, using the <literal>min</literal> attribute:</para>
```

Comment:
only for the "should" modus

Reply:
Also, typo: `minium`

Reply:
@dizzzz So min only works as an attribute on `<bool>` when ALL of its child elements have the `@occur="should"` attribute? That’s not mentioned in https://github.com/eXist-db/exist/issues/392, where the `@occur` attribute isn’t on any of the child elements. In that case of omitted `@occur` attributes, are they implied?

Reply:
Yes I’m confused about the should thing too now.

Reply:
it works on the `bool` element, eg `<bool min="4">` as in the test

Reply:
so it wouldn't work with `must` or `not` ?

Reply:
correct (according to the Lucene javadoc)

Reply:
refs:
javadoc: https://lucene.apache.org/core/3_0_3/api/core/org/apache/lucene/search/BooleanQuery.html#setMinimumNumberShouldMatch(int) 
code: https://www.cc.gatech.edu/computing/pag/tmp/libanalysis/typestate/lusearch/org/apache/lucene/search/BooleanQuery.java.html

Reply:
![image](https://user-images.githubusercontent.com/1700062/68940935-c9ce6880-07a4-11ea-87ec-f8eb550a06c9.png)

Reply:
@dizzzz The link to the javadocs is helpful. Here's the link to the javadocs corresponding to the version we're currently using in eXist, Lucene 4.10.4: https://lucene.apache.org/core/4_10_4/core/org/apache/lucene/search/BooleanQuery.html#setMinimumNumberShouldMatch(int)

Reply:
ok i ll add that as not to this PR to make it more clear.